src/components/TradingViewChart/datafeed.tsx

```diff
@@ -54,6 +54,32 @@ const getNetworkString = (chainId: ChainId | undefined) => {
   }
 }
 
+const DextoolSearchV2ChainId: { [chain in ChainId]: string } = {
```

Comment:
consider using `NETWORK_INFO[chainId].route`

Reply:
its different as dextool naming their own way for each api, this list is not complete yet, I will update for all chain tomorrow